### ISSUE TITLE
Very low enchantment weapons generated occasionally

### ISSUE THREAD

**Issue by AlexMooney** [Attnam/ivan#204]
Sometimes I see critters with weapons that are so negative, they make the display of the weapon screw up.

![image](https://cloud.githubusercontent.com/assets/8269566/19630309/1a6f09fe-9956-11e6-9236-3c50cf86c956.png)

**Comment by andrewtweber** [comment]
Do you think we should allow -5 enchantment weapons to appear, but just ensure that the damage is never < 0?

**Comment by AlexMooney** [comment]
I suspect that the generation of -5 weapons is a bug.  If it's not a bug, then we should probably just make the damage 0 and have the AI drop them.

**Comment by AlexMooney** [comment]
Also happens with armor.

![image](https://cloud.githubusercontent.com/assets/8269566/19633173/9c13799c-997c-11e6-9fbc-663aeab5de3b.png)

**Comment by emlai** [comment]
Is this actually two separate bug reports?
- generation of -5 items
- negative damage value

**Comment by AlexMooney** [comment]
This report is about the generation of really really terrible equipment; I've opened https://github.com/Attnam/ivan/issues/206 to track the negative damage problems.

**Comment by fejoa** [comment]
Only place I can see where this happens in the code is in here: https://github.com/Attnam/ivan/blob/master/Main/Source/gear.cpp#L1057

Which is called during monster generation. There are two factors that increase the chance the item will spawn with negative enchantment: One is the current level's ``EnchantmentMinusChance``, and the other is the item's inherent ``EnchantmentMinusChance``.

I think we should let negatively enchanted items spawn, but fix up the negative damage.

**Comment by AlexMooney** [comment]
I think we should just destroy (or prevent) any weapon generated that's so useless, but then again I'm not rolling up my sleeves and doing it... :stuck_out_tongue_closed_eyes:

**Comment by andrewtweber** [comment]
> I think we should let negatively enchanted items spawn, but fix up the negative damage.

I agree with that. I would vote to close this one and fix the negative damage in the other one

**Comment by red-kangaroo** [comment]
This came up at the forums again, this time because of unique monsters that sometimes get generated with all their gear heavily disenchanted. Maybe we should block at least uniques from getting negatively enchanted gear?
